adafruit_thermal_printer/thermal_printer_legacy.py

```diff
@@ -109,7 +109,7 @@ def feed(self, lines):
         """Advance paper by specified number of blank lines."""
         # Just send line feeds for older printers.
         for _ in range(lines):
-            self._write_char("\n")
+            self._write_char("\n", encoding="ascii")
```

Comment:
We don't need to specify the `ascii` encoding here.

Reply:
It was necessary when I made the `encoding` argument mandatory. I undid this change and removed the explicit encoding from there. It is now encoding via "utf-8" which does not make a difference for the character `"\n"`.